lib/plug/error_handler.ex

```diff
@@ -41,12 +42,22 @@ defmodule Plug.ErrorHandler do
   **Note:** If this module is used with `Plug.Debugger`, it must be used
   after `Plug.Debugger`.
   """
+
+  @doc """
+  Handle errors from plugs.
+
+  Called when an exception is raised during the processing of a plug.
+  """
+  @callback handle_errors(Plug.Conn.t(), map()) :: no_return()
```

Comment:
Can we type the map too? It is kind: :error | :throw | :exit, then exception | term, and Exception.stacktrace.

Reply:
Done!